### ISSUE TITLE
Screen Readers: Many Pages: The listed options are announced without a button role

### ISSUE THREAD

**Issue by olenamihno** [Expensify/App#77558]
## Action Performed:
Precondition: The User should be logged in

1. Using iOS VoiceOver or Android TalkBack, open the Expensify app
2. Start the onboarding flow
3. Swipe through the 'What do you want to do today?' options
4. Note how the options are announced.

## Expected Result:
Each option should be announced with a button role.

## Actual Result:
The listed options are announced without a button role.

## Area issue was found in:
Onboarding

## Failed WCAG checkpoints
WCAG 4.1.2 Name, Role, Value
https://www.w3.org/WAI/WCAG22/Understanding/name-role-value

## User impact:
Providing role, state, and value information on all user interface components enables compatibility with assistive technology, such as screen readers, screen magnifiers, and speech recognition software, used by people with disabilities.

## Suggested resolution:
Apply the accessibilityRole="button" attribute to each listed option.
https://reactnative.dev/docs/button

## Environment:
Apple iPhone 11 - iOS 17.5.1, H0T Mobile - Native (No mobile browser)

## Screenshots/Videos

<img width="828" height="1792" alt="Image" src="https://github.com/user-attachments/assets/d1c019cb-f4d7-499d-b155-a3b635f0808d" />

https://github.com/user-attachments/assets/946ecef0-0e21-4e3f-9a62-5b022faba8a0

**Version Number:** Production
**Reproducible in staging?:** NA
**Reproducible in production?:** Yes
**Email or phone of affected tester (no customers):** No
**Logs:** https://stackoverflow.com/c/expensify/questions/4856  NA
**Notes/Photos/Videos:** Any additional supporting documentation  Yes
**Expensify/Expensify Issue URL:** https://platform.applause.com/company/12102/products/36628/community-issues/7005676
**Issue reported by:** NA
**Slack conversation:** NA
